src/config-util/getConfig.js

```diff
@@ -0,0 +1,95 @@
+'use strict'
+
+const _ = require('lodash')
```

Comment:
Let's try to keep the dependency to lodash at minimum and not use the namespace. It will make future refactorings easier. Ideally, we won't be using lodash at all since it's way too big for the project IMO.